### DISCUSSION TITLE
how to officially released game

### DISCUSSION THREAD

**Discussion by BryanZhu-Us** [UPBGE/upbge#1707] [Ideas/Features for UPBGE 0.3+]
Recently, I have been studying upbge and experimenting with upbge3.3. I want to know that pass the “save as game runtime” release game will include the project source file, that's not safe for the officially released game project. Will there be a better function to release the game in the future? About what time?
thanks!

**Comment by HolandonHdunto** [comment]
I think in upbge 0.2.x you can corrupt .blend files on purpose, changing .blend to .data for example, or .beta, .geometry,.entity and so on. When you click on it, it doesn't open and if someone tries to open it using the directory it will get confused, the only file that can be opened is the executable.

  To edit later is very simple and just open upbge and open the file corrupted by upbge.

Basically you just need a .blend file in which you will export the game, which also opens another corrupted .blend file where the game actually is, so in the end only the executable and the corrupt file are left.



https://user-images.githubusercontent.com/89164699/173249981-e7da289b-061c-48c1-9e39-8186f000e49a.mp4



Of course it is good to try to do this with a copy and not with the original.

**Reply by BryanZhu-Us** [reply]
Thank you. 
Actually, the version I currently use is 0.33. I don't really want to go back to 0.2.x version

**Comment by HolandonHdunto** [comment]
Also works on upbge 0.3.

**Comment by BryanZhu-Us** [comment]
Thank you for this compromise case,I will try it now
